js/filter.js

```diff
@@ -0,0 +1,85 @@
+const mapFiltersForm = document.querySelector('.map__filters');
+const typeField = mapFiltersForm.querySelector('#housing-type');
+const priceField = document.querySelector('#housing-price');
+const roomsField = document.querySelector('#housing-rooms');
+const guestField = document.querySelector('#housing-guests');
+
+const FILTER_PRICE = {
+  low: 10000,
+  high: 50000,
+};
+
+
+const filterType = (item) => {
+  if (item.offer.type === typeField.value) {
+    return item;
+  }
+};
+
+const filterPrice = (item) => {
+  switch (priceField.value) {
+    case 'low': return item.offer.price <= FILTER_PRICE.low;
```

Comment:
```suggestion
    case 'low': {
       return item.offer.price <= FILTER_PRICE.low;
    }
```

По критериям обязательно использовать блоки кода